core/runtime/src/main/java/io/quarkus/runtime/logging/CategoryConfig.java

```diff
@@ -1,20 +1,34 @@
 package io.quarkus.runtime.logging;
 
+import io.quarkus.runtime.annotations.ConfigDocSection;
 import io.quarkus.runtime.annotations.ConfigGroup;
 import io.quarkus.runtime.annotations.ConfigItem;
 
 @ConfigGroup
 public class CategoryConfig {
 
     /**
-     * The minimum level that this category can be set to
+     * The log level level for this category
      */
     @ConfigItem(defaultValue = "inherit")
-    String minLevel;
```

Comment:
This should be removed in a separate PR.

Reply:
Done, pull request https://github.com/quarkusio/quarkus/pull/5894 is made for this change.